### PR TITLE
WA pact tests updated to resolve data mismatch on provider side tests…

### PR DESCRIPTION
**Pull request by sreekanthpuligadda**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-4250 

### Change description ###
Updates to WA PACT tests,
Update to mutation test config. 

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

File: api/test/pact/pact-tests/settings/provider.mock.ts [hunk 1]
```diff
@@ -15,7 +15,7 @@ export class PactTestSetup {
     this.provider = new Pact({
       consumer: "xui_webApp",
       dir: path.resolve(process.cwd(), "api/test/pact/pacts"),
-      log: path.resolve(process.cwd(), "api/test/pact/logs", "mockserver-integration.log"),
+      log: path.resolve(process.cwd(), "api/test/pact/logs", config.provider+".log"),
```
Comment (phillip-whitaker) [inline api/test/pact/pact-tests/settings/provider.mock.ts:18]: Are you not missing a double quote at the start of config.provider ?

Comment (phillip-whitaker) [reply api/test/pact/pact-tests/settings/provider.mock.ts:18]: Also, other parts of the project use single quotes. Should be consistent.

File: stryker.node.conf.js [hunk 2]
```diff
@@ -1,9 +1,27 @@
...
+    "caseshare",
+    "health","healthCheck",
+    "lib",
+    "noc",
+    "organisations",
+    "searchCases","services",
+    "termsAndConditions",
+    "user","userTermsAndConditions",
+    "workAllocation"
+
```
Comment (phillip-whitaker) [inline stryker.node.conf.js:14]: remove blank space

**Review by phillip-whitaker — CHANGES_REQUESTED** [review]
A couple of comments

File: api/test/pact/pact-tests/settings/provider.mock.ts (hunk 1 above)
Comment (phillip-whitaker) [reply api/test/pact/pact-tests/settings/provider.mock.ts:18]: Just seen what is going on here - please use string interpolation instead

```suggestion
      log: path.resolve(process.cwd(), "api/test/pact/logs", `${config.provider}.log`),
```

**Review by Vamshi-Muniganti — APPROVED** [review]
looks fine to me

File: api/test/pact/pact-tests/settings/provider.mock.ts (hunk 1 above)
Comment (udaydenduluri33) [reply api/test/pact/pact-tests/settings/provider.mock.ts]: @phillip-whitaker is no longer in the team. And I can see that its addressed

**Review by udaydenduluri33 — APPROVED** [review]
I can see that PACTS are now running which itself means that this code changes are working. @sreekanthpuligadda  LGTMs

**Review by Mo-Lala84 — APPROVED** [review]
approved now pact tests now resolving the mismatch

**Pull request merged**